DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -21,12 +22,14 @@ public IActionResult Index()
                 return RedirectToAction("Index", "Login");
             }
 
-            var userEmail = User.GetEmail();
+            var userEmail = User.GetCustomClaim(ClaimTypes.Email);
             var delegateId = User.GetCustomClaim(CustomClaimTypes.LearnCandidateNumber);
             var centreId = User.GetCentreId();
+            var firstName = User.GetCustomClaim(CustomClaimTypes.UserForename);
+            var surname = User.GetCustomClaim(CustomClaimTypes.UserSurname);

```

Comment:
Thinking about future features, I think we'll need to take these values from the database, because I don't think it makes sense to update the claims on the cookie when the user edits these details on a different page.

I will need to check what the old code does and whether it's something we need to handle in old pages.

Reply:
[note that the delegate id and centre id are not editable to my knowledge]

Reply:
Yeah I figured that might be the case, I just followed what was already there for now. I need to add some database access for the other 2 parts of the ticket.

Reply:
Cool - I'm happy for you to change this to use the data from db as a separate subtask of the story. :)